packages/lpa-questionnaire-web-app/src/lib/file-upload-helpers.js

```diff
@@ -13,20 +13,111 @@ exports.MIME_TYPE_TIF = 'image/tiff';
 exports.MIME_TYPE_JPEG = 'image/jpeg';
 exports.MIME_TYPE_PNG = 'image/png';
 
-exports.getErrorHtml = (error) => {
+/**
+ *
+ * @param name name or ID of file to delete
+ * @param req request sent
+ */
+exports.deleteFile = (name, req) => {
+  if (!name || !req) return;
```

Comment:
Should this return some kind of error for the try/catch block in upload-plans.js?

Reply:
The error mainly for missing delete file